src/registration.html

```diff
@@ -1,76 +1,92 @@
 <!DOCTYPE html>
 
-<html>
+<html class="fullsize">
     <head>
         <link rel="stylesheet" href="registration.css">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;700;900&display=swap" rel="stylesheet"> 
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     </head>
-    <body>
+    <body class="fullsize">
         <header>
-            vinyl exchange
+            <span class="material-icons">
+                <span class ="magenta">disc_full</span>
+            </span>
+            vinyl<span class ="magenta">exchange</span>
         </header>
-        <main>
+        <main class="fullsize">
             <form method="POST">
-                <div>
+                <div class="pad">
                     <label>
-                        First Name <input id="firstname" name="fname" type="text" required/>
+                        First Name 
+                        <p><input id="firstname" name="fname" class="inputstyle" type="text" required/></p>
                     </label>
                 </div>
-                <div>
+                <div class="pad">
                     <label>
-                        Last Name <input id="lastname" name="lname" type="text" required/>
+                        Last Name 
+                        <p><input id="lastname" name="lname" class="inputstyle" type="text" required/></p>
                     </label>
                 </div>
-                <div>
+                <div class="pad">
+                    <p>
```

Comment:
Remove the `<p>` tag, as it doesn't do anything in this case, and even if it would, a paragraph element should only be used when separating actual text paragraphs.